src/pages/workspace/reimburse/WorkspaceReimburseSection.tsx

```diff
@@ -1,45 +1,39 @@
-import lodashGet from 'lodash/get';
-import PropTypes from 'prop-types';
 import React, {useEffect, useState} from 'react';
 import {ActivityIndicator, View} from 'react-native';
+import type {OnyxEntry} from 'react-native-onyx';
 import ConnectBankAccountButton from '@components/ConnectBankAccountButton';
 import * as Expensicons from '@components/Icon/Expensicons';
 import * as Illustrations from '@components/Icon/Illustrations';
-import networkPropTypes from '@components/networkPropTypes';
 import Section from '@components/Section';
 import Text from '@components/Text';
+import useLocalize from '@hooks/useLocalize';
+import useNetwork from '@hooks/useNetwork';
 import usePrevious from '@hooks/usePrevious';
 import useTheme from '@hooks/useTheme';
 import useThemeStyles from '@hooks/useThemeStyles';
 import BankAccount from '@libs/models/BankAccount';
-import * as ReimbursementAccountProps from '@pages/ReimbursementAccount/reimbursementAccountPropTypes';
 import * as Link from '@userActions/Link';
+import type * as OnyxTypes from '@src/types/onyx';
 
-const propTypes = {
+type WorkspaceReimburseSectionProps = {
     /** Policy values needed in the component */
-    policy: PropTypes.shape({
-        id: PropTypes.string,
-    }).isRequired,
+    policy: OnyxEntry<OnyxTypes.Policy>;
 
     /** Bank account attached to free plan */
-    reimbursementAccount: ReimbursementAccountProps.reimbursementAccountPropTypes.isRequired,
-
-    /** Information about the network */
-    network: networkPropTypes.isRequired,
-
-    /** Returns translated string for given locale and phrase */
-    translate: PropTypes.func.isRequired,
+    reimbursementAccount: OnyxEntry<OnyxTypes.ReimbursementAccount>;
 };
 
-function WorkspaceReimburseSection(props) {
+function WorkspaceReimburseSection({policy, reimbursementAccount}: WorkspaceReimburseSectionProps) {
     const theme = useTheme();
     const styles = useThemeStyles();
+    const {translate} = useLocalize();
     const [shouldShowLoadingSpinner, setShouldShowLoadingSpinner] = useState(true);
-    const achState = lodashGet(props.reimbursementAccount, 'achData.state', '');
+    const achState = reimbursementAccount?.achData?.state ?? '';
     const hasVBA = achState === BankAccount.STATE.OPEN;
-    const reimburseReceiptsUrl = `reports?policyID=${props.policy.id}&from=all&type=expense&showStates=Archived&isAdvancedFilterMode=true`;
-    const isLoading = lodashGet(props.reimbursementAccount, 'isLoading', false);
+    const reimburseReceiptsUrl = `reports?policyID=${policy?.id ?? ''}&from=all&type=expense&showStates=Archived&isAdvancedFilterMode=true`;
```

Comment:
Can we extract this url to `CONST`?

Reply:
Do you mean add this into the CONST file?